go.mod

```diff
@@ -176,3 +176,5 @@ require (
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 )
+
+replace github.com/imdario/mergo => github.com/imdario/mergo v0.3.16
```

Comment:
Could you please add an explanation (or better yet an open issue if there is any) why the replace is here and when it could be removed?